src/libraries/System.Runtime.InteropServices/tests/System/Runtime/InteropServices/PosixSignalRegistrationTests.Unix.cs

```diff
@@ -57,7 +59,15 @@ public void SignalHandlerCalledForKnownSignals(PosixSignal s)
                     semaphore.Release();
                 });
 
-                kill(signal);
+                // Use 'kill' executable with signal name to validate the signal pal mapping.
```

Comment:
How does this validate the mapping more than it did previously?

Reply:
Previously we were relying on our own mapping to go from enum value to platform value here:

https://github.com/dotnet/runtime/blob/7b45670ed33d80c54decb855d728c3ba17a61d3d/src/libraries/System.Runtime.InteropServices/tests/System/Runtime/InteropServices/PosixSignalRegistrationTests.Unix.cs#L221-L227

Now, we're using `kill` to translate the enum value name (e.g. `SIGINT`) to the platform value.

Reply:
Thanks.  I missed this was using the name rather than the value.